ui/app/AppLayouts/Communities/views/EditAirdropView.qml

```diff
@@ -512,27 +513,34 @@ StatusScrollView {
                 model: SortFilterProxyModel {
                     sourceModel: membersModel
 
+                    sorters : [
+                        StringSorter {
+                            roleName: "preferredDisplayName"
+                            caseSensitivity: Qt.CaseInsensitive
+                        }
+                    ]
+
                     filters: [
-                        ExpressionFilter {
+                        FastExpressionFilter {
                             enabled: membersDropdown.searchText !== ""
 
                             function matchesAlias(name, filter) {
                                 return name.split(" ").some(p => p.startsWith(filter))
                             }
 
                             expression: {
-                                membersDropdown.searchText
-
                                 const filter = membersDropdown.searchText.toLowerCase()
                                 return matchesAlias(model.alias.toLowerCase(), filter)
                                          || model.displayName.toLowerCase().includes(filter)
                                          || model.ensName.toLowerCase().includes(filter)
                                          || model.localNickname.toLowerCase().includes(filter)
                                          || model.pubKey.toLowerCase().includes(filter)
                             }
+                            expectedRoles: ["alias", "displayName", "ensName", "localNickname", "pubKey"]
                         },
-                        ExpressionFilter {
+                        FastExpressionFilter {
                             expression: !!model.airdropAddress
+                            expectedRoles: ["airdropAddress"]
                         }
```

Comment:
If it's about check if `airdropAddress` is not empty string, `ValueFilter` could be used:
```qml
ValueFilter {
  roleName: "airdropAddress"
  value: ""
  inverted: true
}
```